asmcli/cloudbuild-main.yaml

```diff
@@ -6,17 +6,22 @@ steps:
   - name: 'gcr.io/cloud-builders/bazel'
     entrypoint: 'bazel'
     args: ['build', 'merge']
+  
   - name: 'bash'
     args: ['cp', '${_BAZEL_BIN}/${_SCRIPT_NAME}', asmcli/]
```

Comment:
I think you're already aware, but we should remove this once we figure out a way to get the artifact committed. No action on this PR, just mentioning it.

Reply:
Will do.